frontend/src/main/components/Jobs/JobsTable.js

```diff
@@ -1,8 +1,6 @@
 import React from "react";
-import OurTable, {
-  PlaintextColumn,
-  DateColumn,
-} from "main/components/OurTable";
+import OurTable, { DateColumn } from "main/components/OurTable";
```

Comment:
Goes with later suggestion

```suggestion
import OurTable, { DateColumn } from "main/components/OurTable";
import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
```

Reply:
I understand why AdminJobLogPage needs BasicLayout, but does JobsTable need it as well? I checked the other files within the same folder as this file and none of them use BasicLayout.

Reply:
No it doesn't... may have put this suggestion in the wrong file.  Sorry about that.